<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupWindow</class>
 <widget class="QMainWindow" name="signupWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>391</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>DejaVu Sans</family>
      <pointsize>30</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	background-color: #faedcd;
}</string>
    </property>
    <property name="text">
     <string>       Sign up</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #faedcd;
	border: 3px solid green;
	border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="surnameLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #faedcd;
	border: 3px solid green;
	border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="repeatLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>550</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #faedcd;
	border: 3px solid green;
	border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="passwordLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #faedcd;
	border: 3px solid green;
	border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="emailLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>18</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: #faedcd;
	border: 3px solid green;
	border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signupbtn">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>640</y>
      <width>231</width>
      <height>65</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sans</family>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #faedcd;
	border: 3px solid green;
	border-radius: 20px;
}

QPushButton:hover {
	background-color: #ffffff;
}</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
